Client/src/ClientSceneManager/ClientSceneManager.hpp

```diff
@@ -0,0 +1,88 @@
+/*
+** EPITECH PROJECT, 2024
+** R-Type
+** File description:
+** ClientSceneManager
+*/
+
+#pragma once
+
+#include <unordered_map>
+
+#include <jsoncpp/json/json.h>
```

Comment:
We have to discuss about this include. Because between Ubuntu and Fedora, the include is not the same.
So what do we do ?

Reply:
Perhaps the package manager will resolve that question...(hope)

Reply:
Let's wait for @theophile-jr's PR (the package manager). Then, if it still doesn't work, let find a solution to this issue

Reply:
Seen with Théophile